tools/project_manager/displayimageonly.php

```diff
@@ -0,0 +1,40 @@
+<?php
+$relPath='../../pinc/';
+include_once($relPath.'base.inc');
+include_once($relPath.'misc.inc'); // get_enumerated_param()
+include_once($relPath.'slim_header.inc');
+include_once($relPath.'page_controls.inc'); // get_page_data_js()
+
+require_login();
+
+$error_message = "";
+$projectid = "";
+$imagefile = "";
+try
+{
+    $projectid = get_projectID_param($_GET, 'project', true);
+    $imagefile = get_page_image_param($_GET, 'imagefile', true);
+}
+catch(Exception $exception)
+{
+    $error_message = $exception->getMessage();
+}
+
+$title = _("Display Image for page");
+
+$js_files = [
+    "$code_url/scripts/page_browse.js",
+    "$code_url/tools/project_manager/display_image_only.js",
+    ];
```

Comment:
Outdent this so it matches the `$header_args` def.

Reply:
done this